docs/Getting Started/Adapting to the Head Unit Language/index.md

```diff
@@ -2,11 +2,12 @@
 Since a head unit can support multiple languages, you may want to add support for more than one language to your SDL app. The SDL library allows you to check which language is currently be used by the head unit. If desired, the app's name and the app's text-to-speech (TTS) name can be customized to reflect the head unit's current language. If your app name is not part of the current lexicon, you should tell the VR system how a native speaker will pronounce your app name by setting the TTS name using [phonemes](https://en.wikipedia.org/wiki/Phoneme) from either the Microsoft SAPI phoneme set or from the LHPLUS phoneme set.
 
 ## Setting the Default Language
-The initial configuration of the `SDLManager` requires a default language when setting the `SDLLifecycleConfiguration`. If not set, the SDL library uses American English (*EN_US*) as the default language. The connection will fail if the head unit does not support the `language` set in the `SDLLifecycleConfiguration`. The `RegisterAppInterfaceResponse` RPC will return `INVALID_DATA` as the reason for rejecting the request.
+The initial configuration of the @![iOS]`SDLManager`!@@![android,javaSE,javaEE]`SdlManager`!@ requires a default language when setting the @![iOS]`SDLLifecycleConfiguration`!@@![android,javaSE,javaEE]`Builder`!@. If not set, the SDL library uses American English (*EN_US*) as the default language. The connection will fail if the head unit does not support the `language` set in the @![iOS]`SDLLifecycleConfiguration`!@@![android,javaSE,javaEE]`Builder`!@. @![iOS]The `RegisterAppInterfaceResponse` RPC will return `INVALID_DATA` as the reason for rejecting the request!@ @![android,javaSE,javaEE] `//TODO Android to add what happens in case of unsupported language`!@.
```

Comment:
This is generic. You can remove the platform tags and TODO from:
@![iOS]The `RegisterAppInterfaceResponse` RPC will return `INVALID_DATA` as the reason for rejecting the request!@ @![android,javaSE,javaEE] `//TODO Android to add what happens in case of unsupported language`!@.
AND
Change `RegisterAppInterfaceResponse` to `RegisterAppInterface` response